### DISCUSSION TITLE
PROJECT_BUILD_NUMBER is 1 if new workflow is created

### DISCUSSION THREAD

**Discussion by zuboje** [codemagic-ci-cd/codemagic-docs#1326] [Q&A] [answered]
I have a workflow that I built some time ago and have 99 builds. I just built a brand-new workflow and when using a variable $PROJECT_BUILD_NUMBER in my new workflow it returned 1, while it should have returned 100. Do personal accounts have the ability to do this or "project" is only when you have a team feature?

**Comment by himesh-cm** [comment, marked as answer]
Hi @zuboje, You can find the list of built-in variables [here](https://docs.codemagic.io/yaml-basic-configuration/environment-variables/). Also, these are generic platform features available for everyone.
I think you are looking at `$BUILD_NUMBER` - The number of the build for this project in Codemagic for the given workflow.

**Reply by zuboje** [reply]
> Hi @zuboje, You can find the list of built-in variables [here](https://docs.codemagic.io/yaml-basic-configuration/environment-variables/). Also, these are generic platform features available for everyone. I think you are looking at `$BUILD_NUMBER` - The number of the build for this project in Codemagic for the given workflow.

BUILD_NUMBER - Number of the build for this project in Codemagic for the given workflow
PROJECT_BUILD_NUMBER - Number of the build for this project in Codemagic

BUILD_NUMBER is unique for a particular flow, but PROJECT_BUILD_NUMBER should count all flows together according to the documentation. So for example, if I have two flows, flow 1 has 54 builds, and flow 2 has 12 builds this means:

next flow 1 BUILD_NUMBER is 55, PROJECT_BUILD_NUMBER 67
next flow 2 BUILD_NUMBER is 13, PROJECT_BUILD_NUMBER 67

This is in the documentation but I am not seeing this being true in my workflows, Here is an example from my flows

my workflow 1 build:

build arguments `--build-name=2.20.$(($BUILD_NUMBER - 96)) --build-number=$(($PROJECT_BUILD_NUMBER + 254))`

my workflow 1 result:

`flutter build ipa --release --export-options-plist /Users/builder/export_options.plist --build-name=2.20.3 --build-number=353`

**This indicates that my BUILD_NUMBER = 99, PROJECT_BUILD_NUMBER = 99**

my workflow 2 build:

build arguments `--build-name=2.20.$(($BUILD_NUMBER + 2)) --build-number=$(($PROJECT_BUILD_NUMBER + 254))`

my workflow 2 result (that I've just built):

`flutter build ipa --release --export-options-plist /Users/builder/export_options.plist --build-name=2.20.3 --build-number=255`

**This indicates that my BUILD_NUMBER = 1, PROJECT_BUILD_NUMBER = 1**

While I expcet BUILD_NUMBER in second workflow to be 1, PROJECT_BUILD_NUMBER should be 100

**Reply by himesh-cm** [reply]
Can you please share the build id as well?

**Reply by zuboje** [reply]
63c7480c58286fd092ac6148

**Reply by zuboje** [reply]
@himesh-cm I figured out my issue I had two different apps instead of creating two different workflows within the same app
